### PR TITLE
ryan attempts to clean up code for cow checklist (gold)

### PR DESCRIPTION
**Pull request by ryanchou-dev**
please don't fire me 🙏

---
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: solutions/usaco-670.mdx [hunk 1]
```diff
@@ -2,89 +2,107 @@
...
-{
-	int dx = a.first - b.first, dy = a.second - b.second;
-	return dx * dx + dy * dy;
+using ll = long long;
+
+struct coord {
+	int x, y;
+};
+
+ll dist(coord &a, coord &b) {
```
Comment (SansPapyrus683) [inline solutions/usaco-670.mdx:38]: add `const`

File: solutions/usaco-670.mdx [hunk 2]
```diff
@@ -2,89 +2,107 @@
...
+#include <vector>
 using namespace std;
 
-int energy(pair<int, int> a, pair<int, int> b)
-{
-	int dx = a.first - b.first, dy = a.second - b.second;
-	return dx * dx + dy * dy;
+using ll = long long;
+
+struct coord {
```
Comment (SansPapyrus683) [inline solutions/usaco-670.mdx:34]: `Coord` instead

File: solutions/usaco-670.mdx [hunk 3]
```diff
@@ -2,89 +2,107 @@
...
+	int x, y;
```
Comment (SansPapyrus683) [inline solutions/usaco-670.mdx:35]: explicitly declare `public`

File: solutions/usaco-670.mdx [hunk 4]
```diff
@@ -2,89 +2,107 @@
...
+	/*
+	 * min_dist[i][j][k] represents the minimum distance for
+	 * Farmer John to visit the first i Holsteins and the first
+	 * j Guernseys, given that Farmer John is currently at
+	 * k (0: ith Holstein, 1: ith Guernsey)
+	 */
+
+	// set default value to arbitrarily large integer to prevent overflow
+	vector<vector<vector<ll>>> min_dist(1001, 
+	vector<vector<ll>>(1001, vector<ll>(2, 1e18)));
```
Comment (SansPapyrus683) [inline solutions/usaco-670.mdx:71]: magic #'s

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/usaco-670.mdx (hunk 3 above)
Comment (envyaims) [reply solutions/usaco-670.mdx:35]: ??

**Review by envyaims — APPROVED** [review]
aside from what sans already pointed out, seems great :blobheart:

File: solutions/usaco-670.mdx (hunk 1 above)
Comment (envyaims) [reply solutions/usaco-670.mdx:38]: wot

File: solutions/usaco-670.mdx (hunk 3 above)
Comment (SansPapyrus683) [reply solutions/usaco-670.mdx:35]: explicit better than implicit

File: solutions/usaco-670.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/usaco-670.mdx:38]: yeah, but it's not like the function needs a copy or anything

File: solutions/usaco-670.mdx [hunk 5]
```diff
@@ -2,89 +2,108 @@
...
+	/*
+	 * min_dist[i][j][k] represents the minimum distance for
+	 * Farmer John to visit the first i Holsteins and the first
+	 * j Guernseys, given that Farmer John is currently at
+	 * k (0: ith Holstein, 1: ith Guernsey)
+	 */
+
+	// set default value to arbitrarily large integer to prevent overflow
+	vector<vector<vector<ll>>> min_dist(n + 1, 
+	vector<vector<ll>>(n + 1, vector<ll>(2, MAX_D)));
```
Comment (SansPapyrus683) [inline solutions/usaco-670.mdx:72]:
```suggestion
	vector<vector<vector<ll>>> min_dist(
		n + 1, vector<vector<ll>>(n + 1, vector<ll>(2, MAX_D))
	);

**Review by SansPapyrus683 — APPROVED** [review]

**Pull request merged**